static/scss/partials/main.scss

```diff
@@ -2101,6 +2106,16 @@ input.input-has-error {
   }
 }
 
+@media screen and (max-width: 660px) {
+  .hide-mobile-link {
+    display: none !important;
+  }
+
+  .show-mobile-link {
+    display: flex !important;
```

Comment:
Are the `!important` tags necessary?

Reply:
The original `.hide-mobile` class had it so I just carried it over - should I remove it?

Reply:
Yeah. `!important` should be super super rare.

Reply:
Noted. I removed both `!important`s in my last commit.